tests/run-pass/conf_whitelisted.rs

```diff
@@ -1,4 +0,0 @@
-#![feature(plugin)]
-#![plugin(clippy(conf_file="./tests/auxiliary/conf_whitelisted.toml"))]
```

Comment:
Is overriding the conf file path not supported anymore?

Reply:
Still supported. Just not testable anymore... actually. .. run pass could be a regular rust integration test. I'll check